minos/networks/__init__.py

```diff
@@ -26,6 +26,9 @@
     MinosNetworkException,
 )
 from .handlers import (
+    BrokerCommandEnroute,
+    BrokerEventEnroute,
+    BrokerQueryEnroute,
```

Comment:
In this case I think it's a better choice to only expose the `enroute` variable and keep hidden all the auxiliary classes to the framework users

Reply:
Done!